rakelib/kokoro/kokoro.rb

```diff
@@ -237,6 +243,14 @@ def run_ci gem = nil, local = false
     verify_in_gemfile gem unless local
   end
 
+  def updated_samples
+    updated_directories = `git --no-pager diff --name-only HEAD^ HEAD | grep "/" | cut -d/ -f 1-2 | sort | uniq || true`
+    updated_directories = updated_directories.split "\n"
+    updated_directories.select! { |dir| dir.include? "samples" }
```

Comment:
I'd be a bit more cautious, in case there's another directory with the name "samples". Maybe something like

```ruby
updated_directories.select! { |dir| dir.split("/")[1] == "samples" }
```